index.md

```diff
@@ -1,25 +1,224 @@
+let us know if you have any feedback or suggestions!
+This is what you will learn in this CodeLab:
+
+### 1. Creating a new project with our Android Core Library
+
+We have our own Android Core Library that we use in all our projects. This library contains a lot of useful classes
+and functions that we use in all our projects.
+
+### 2. Folder structure
+
+We have a specific folder structure that we use in all our projects. This is to make sure that we can easily find
+all the files we need.
+
+### 3. AGILE
+
+1. Work with **Jira**
+2. Using **Bitbucket** for version control
+3. Branching strategy
+4. **Pull Requests**
+
+### 4. Creating a first screen
+
+* How to use **Figma**
+* Write some code!
+* Write some more code!
+* Create your first **PR** (pull request)
+
+### Let's get started!
+
+![](index/img/intro/android-hello.gif)
+
+## What you need: Prerequisites
+
+Duration: 0:55:00
+
+Before we start, make sure you have everything you need to complete this **WiseLab**. You will need:
+
+### Android Studio
+
+Download the IDE here if you don't have it already!
+[Android Studio](https://developer.android.com/studio)
+
+### Figma
+
+Our designers work with **Figma**. You can download it here:
+
+* [Figma](https://www.figma.com/downloads/)
+
+To access the designs you need to log in with your Wisemen account:
+
+* [Figma wireframes](https://www.figma.com/file/hebgv4Qx8VanMAQkO1NFpa/Onboarding-to-do?node-id=407-4095&t=2qdyy89lKwN7dFw3-0)
+
+### BitBucket repository
+
+*ToDo: Add link to BitBucket repository*
+
+### Jira access
+
+*ToDo: Add link to Jira*
+
+## Create the project
+
+Duration: 0:50:00
+
+### 3.1 Create a new project
+
+Open Android Studio and create a new project. You can name it Wiselab_Android_YOURNAME.
+
+![](index/img/setup/new_project.png)
+
+### 3.2 Link your project to BitBucket
+
+You can find the link to your repo on your [personal Confluence](https://appwise.atlassian.net/wiki/home) page.
+
+![](index/img/setup/bitbucket_clone.png)
+
+#### Use the terminal or IDE to link your project to BitBucket
+
+We will show you how to work with the ***terminal***. If you prefer to work with the ***IDE***, you can find the
+instructions [here](https://www.jetbrains.com/help/idea/set-up-a-git-repository.html#add-remote).
+
+* Open the terminal in Android Studio (bottom left corner)
+* add remote origin to project
+* ```shell
+  git init
+  git remote add origin <your repo url>
+  ```
+
+From here on you can choose to use the terminal or the IDE to work with Git.
+
+### 3.3 Our Branching strategy
+
+We use the **Gitflow** branching strategy. You can find more information about this
+strategy [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+
+* **master** branch: this branch contains the latest release of the app
+* **staging** branch: this branch is used send to the client for testing
+* **QA** branch: this branch is used to test the features before they are merged into the develop branch
```

Comment:
shouldn't this be:
`**QA** branch: this branch is used to test the features before they are merged into the staging branch`?